acquia_cms.info.yml

```diff
@@ -1,9 +1,12 @@
 name: Acquia CMS
 core_version_requirement: ^9
+version: '1.2.4'
 type: profile
 description: "An opinionated implementation of Drupal 9 for running low code websites in the Acquia platform."
 distribution:
+  name: Acquia CMS
```

Comment:
@chandan-singh7929 Is it ok, if we change our custom profile to distribution ? Because, then this distribution will be installed automatically without asking user to select profile.
cc: @kdruckman

Reply:
@vishalkhode1 - Acquia CMS is already a distribution, this is what its from very beginning.

Reply:
Make sense. Okay, I thought `name` was not added earlier as well (under distribution) intentionally.